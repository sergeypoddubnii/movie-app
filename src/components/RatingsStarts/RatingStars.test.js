import RatingStarts from './RatingsStars';
import React from 'react';
import { shallow } from 'enzyme';

describe('should render RatingsStarts', () => {
  it('RatingsStarts component with props', () => {
    const component = shallow(<RatingStarts rating={5} />);
    expect(component).toMatchSnapshot();
  });
  it('RatingsStarts component without props', () => {
    const component = shallow(<RatingStarts />);
    expect(component).toMatchSnapshot();
  });
});
