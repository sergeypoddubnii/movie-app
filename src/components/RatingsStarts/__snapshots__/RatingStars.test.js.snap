// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render RatingsStarts RatingsStarts component with props 1`] = `
<StarRatings
  changeRating={null}
  gradientPathName=""
  ignoreInlineStyles={false}
  name="rating"
  numberOfStars={10}
  rating={5}
  starDimension="20px"
  starEmptyColor="rgb(203, 211, 227)"
  starHoverColor="rgb(230, 67, 47)"
  starRatedColor="orange"
  starSpacing="0px"
  svgIconPath="m25,1 6,17h18l-14,11 5,17-15-10-15,10 5-17-14-11h18z"
  svgIconViewBox="0 0 51 48"
  typeOfWidget="Star"
/>
`;

exports[`should render RatingsStarts RatingsStarts component without props 1`] = `
<StarRatings
  changeRating={null}
  gradientPathName=""
  ignoreInlineStyles={false}
  name="rating"
  numberOfStars={10}
  rating={0}
  starDimension="20px"
  starEmptyColor="rgb(203, 211, 227)"
  starHoverColor="rgb(230, 67, 47)"
  starRatedColor="orange"
  starSpacing="0px"
  svgIconPath="m25,1 6,17h18l-14,11 5,17-15-10-15,10 5-17-14-11h18z"
  svgIconViewBox="0 0 51 48"
  typeOfWidget="Star"
/>
`;
